packages/mdc-chips/chip-set/foundation.js

```diff
@@ -82,24 +81,25 @@ class MDCChipSetFoundation extends MDCFoundation {
    */
   handleChipInteraction_(evt) {
     const {chip} = evt.detail;
+    const chipFoundation = chip.foundation;
```

Comment:
Reduce these two lines to `const chipFoundation = evt.detail.chip.foundation;` ?